### PR TITLE
DX: use ::class whenever possible - one more class

### PR DESCRIPTION
**Pull request by kubawerlos**

### CODE REVIEW

File: tests/AutoReview/ProjectCodeTest.php [hunk 1]
```diff
@@ -69,7 +69,7 @@ static function ($class) { return !class_exists($class) && !trait_exists($class)
      */
     public function testThatSrcClassHaveTestClass($className)
     {
-        $testClassName = str_replace('PhpCsFixer', 'PhpCsFixer\\Tests', $className).'Test';
+        $testClassName = 'PhpCsFixer\\'.str_replace('PhpCsFixer', 'Tests', $className).'Test';
```
Comment (keradus) [inline tests/AutoReview/ProjectCodeTest.php]: looks wrong

Comment (kubawerlos) [reply tests/AutoReview/ProjectCodeTest.php]: only a little bit :) it was false positive in the bash command - should I add negative lookahead for this case there?

Comment (keradus) [reply tests/AutoReview/ProjectCodeTest.php]: i can discuss lot about "why bash will always give you false positives" for this task, unfortunately :(

File: check_FQCN_strings.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+FQCN_strings=$(
+    find . \
+        -type f \
+        -name '*.php' \
+        -not -path "./.git/*" \
+        -not -path "./dev-tools/bin/*" \
+        -not -path "./dev-tools/vendor/*" \
+        -not -path "./vendor/*" \
+        -not -path "./tests/Fixtures/*" \
+        -exec grep -EIHn "'PhpCsFixer(\\\\{1,2}[a-zA-Z0-9]+)+'" {} \; \
```
Comment (keradus) [inline check_FQCN_strings.sh]: hm... having it only for `PhpCsFixer\Foo` but not for `Symfony\Bar` is kind of killing the purpose of this script for me, like, we will have a lot of not found cases that we need to fix manually, periodically, like nowadays
i'm not happy with current solution (or lack of it), but this one is not making it that better, imho

**Comment by keradus** [comment]
thanks, @kubawerlos

**Pull request merged**